kuma/banners/jinja2/banners/includes/developer-needs.html

```diff
@@ -0,0 +1,15 @@
+<div id="developer-needs" class="mdn-cta-content-container">
+    <h2 class="mdn-cta-title">
+        {{ _('MDN Developer Survey') }}
+    </h2>
+    <p class="mdn-cta-copy">
+        {{ _('We want to know learn more about the needs of developers') }}
```

Comment:
Typo: I suspect you wanted `We want to learn more...` instead of `We want to know learn more...`